deployments/liqo/templates/advertisement-rbac.yaml

```diff
@@ -2,24 +2,24 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: advertisement-operator
+  name: {{ include "generic.service-account" (dict "prefix" (include "advertisement.name" .)) }}
   namespace: {{ .Release.Namespace }}
   labels:
-    k8s-app: advertisement-operator
-    app: liqo.io
+    {{- include "advertisement.selectorLabels" . | nindent 4 }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: advertisement-operator
+  name: {{ include "generic.cluster-role-binding" (dict "prefix" (include "advertisement.name" .)) }}
   labels:
-    k8s-app: advertisement-operator
-    app: liqo.io
+    {{- include "advertisement.selectorLabels" . | nindent 4 }}
 subjects:
   - kind: ServiceAccount
-    name: advertisement-operator
+    name: {{ include "generic.service-account" (dict "prefix" (include "advertisement.name" .)) }}
     namespace: {{ .Release.Namespace }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cluster-admin
\ No newline at end of file
+  name: {{ include "advertisement.name" . }}
+
+{{ (.Files.Get (include "generic.rbac-filename" (dict "prefix" (include "advertisement.name" .)))) }}
```

Comment:
The best to me would be to explicitly define the metadata (to obtain consistency with names and labels), and only include the rules, but I do not know if can be done easiliy (it may be possible to just crop the first lines of the files, but that relies on the structure to be immutable).